portfolio/src/main/webapp/script.js

```diff
@@ -15,15 +15,35 @@
 /**
  * Adds a random greeting to the page.
  */
- function addRandomGreeting() {
-    const greetings =
-        ['Would I rather be feared or loved? Easy. Both. I want people to be afraid of how much they love me.', 'There’s a lot of beauty in ordinary things. Isn’t that kind of the point?', 'Sometimes I’ll start a sentence and I don’t even know where it’s going. I just hope I find it along the way.', 'I guess I’ve been working so hard, I forgot what it’s like to be hardly working.', 'I am Beyonce, always.'];
-  
+
+$(document).ready(function() {
+    $(window).scroll(function() {
+            if (this.scrollY > 20) {
+                $('.navbar').addClass("sticky");
+            } else {
+                $('.navbar').removeClass("sticky");
+            }
+        })
+        //for the menubar
+    $('.menu-bar').click(function() {
```

Comment:
What is this callback doing?

Reply:
oh i was following a media query tutorial for that. I believe that's for the media query for when the navigation fills up the entire screen instead when the window is smaller.

Reply:
This is a click listener. On clicking the menu bar, it is toggling the 'active' css class on elements found by `.navbar .links` and `.menu-bar i`. It's unclear to me what the intent is based on the 'active' css class.

Reply:
@media (max-width: 947px) {
    .menu-bar i.active:before {
        content: "\f00d";
    }
    .navbar .links.active {
        left: 0;
    }

the active class gets called here when the screen gets smaller to change the view of the nav bar

before at full screen:
<img width="1392" alt="Screen Shot 2021-06-06 at 9 56 26 PM" src="https://user-images.githubusercontent.com/62718546/120949367-5be4eb80-c712-11eb-9c83-30e0e4bcf406.png">

after reducing screen:
<img width="705" alt="Screen Shot 2021-06-06 at 9 56 45 PM" src="https://user-images.githubusercontent.com/62718546/120949377-60a99f80-c712-11eb-8315-d39b1fae8461.png">

then when the icon is clicked the active class is added then it looks like this:
<img width="853" alt="Screen Shot 2021-06-06 at 9 57 29 PM" src="https://user-images.githubusercontent.com/62718546/120949401-715a1580-c712-11eb-961c-7b0b51c85800.png">

Reply:
`@media (max-width: 947px) {` this media query certainly does that. But if you remove this click listener, what breaks?

Reply:
i need the click listener to activate the active class because if not if you click/toggle the icon, nothing happens. but because of the listner, when you click the icon it toggles the active class and shows the navigation page

Reply:
It seems like you're talking about two different things. One for the resizing and one for clicking.